src/tool_getparam.c

```diff
@@ -556,7 +556,7 @@ static void cleanarg(argv_item_t str)
 #define cleanarg(x)
 #endif
 
-ParameterError getparameter(const char *flag, /* f or -long-flag */
+ParameterError getparameter(const char *flag, /* the raw argument from argv */
```

Comment:
I don't think this helps and it's not true. For example when the function is used to handle arguments set in .curlrc.

Reply:
Okay, fair. However, the current comment isn't true either, as it has been always `-f` or `--long-flag` for me, but I may have overseen an edge-case.

Reply:
I don't think the fact that it was wrong before is a good reason to change it to another wrong...

How about listing the 4(?) different formats the arguments can be provided?
~~~c
/* "-a", "-ab", "--long" or "long" */

Reply:
sounds good